dspace/config/crosswalks/orcid/mapConverter-orcid-to-dspace-language-code.properties

```diff
@@ -0,0 +1,8 @@
+en = en
```

Comment:
Is this mapping necessary? It looks like the same value is always in the key and in the value?  If it is necessary, I think it needs a comment to describe it's usage.